flare/dft_interface/vasp_util.py

```diff
@@ -8,14 +8,14 @@
 from flare.util import NumpyEncoder
 import os, shutil
 
-def check_vasprun(vasprun: Union[str, Vasprun]) -> Vasprun:
+def check_vasprun(vasprun: Union[str, Vasprun], vasprun_kwargs: dict={}) -> Vasprun:
```

Comment:
Good question-- I do know that one reason to do this is if you have multiple functions downstream which kwargs could feed into, so you can delineate them. It may be a little risk-averse to do it this way,  since this is already a very simple function, but it's probably okay.